stdlib/testing/pandas/extract_regexp_findString_test.flux

```diff
@@ -31,14 +31,9 @@ outData = "
 ,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:06Z,cLnSkNMI,used_percent,disk,disk1,apfs,host.local,/,c
 ,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:54:16Z,13F2,used_percent,disk,disk1,apfs,host.local,/,F
 "
-
 re = regexp.compile(v: "[[:alpha:]]{1}")
+t_string_extract = (table=<-) => table
+    |> range(start: 2018-05-22T19:53:26Z)
+    |> map(fn: (r) => ({r with extract: regexp.findString(r: re, v: r._value)}))
 
-t_string_extract = (table=<-) =>
-	(table
-		|> range(start: 2018-05-22T19:53:26Z)
-		|> map(fn: (r) => ({r with extract: regexp.findString(r: re, v: r._value)}))
-	)
-
-test _string_extract = () =>
-         ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_string_extract})
\ No newline at end of file
+test _string_extract = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_string_extract})
```

Comment:
Now that I've read this a few times, I take it back. I like the `test` being read like this.